PeerioTechnologies/peerio-desktop
# fix: do not allow legacy files to be moved to shared folder

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8150/desktop-sdk-disable-shared-folders-as-a-move-destination-for-pending-files
https://github.com/PeerioTechnologies/peerio-icebear/pull/229
 
#### Testing instructions  
In Files view, when you move a legacy file, it is no longer possible to select a shared folder as the target folder.

This includes
* clicking "move" from the three-dots menu on a legacy file
* clicking "move" from the three-dots menu on a folder that contains a legacy file
* bulk selecting a legacy file
* bulk selecting a folder that contains a legacy file

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
